src/drc/api/filters.py

```diff
@@ -14,7 +49,7 @@ class Meta:
         )
 
 
-class ObjectInformatieObjectFilter(FilterSet):
+class ObjectInformatieObjectFilter(InformatieObjectFilterSet):
```

Comment:
with the proposed changes above, we could turn this into:

```suggestion
from vng_api_common.filters import URLModelChoiceFilter

class ObjectInformatieObjectFilter(FilterSet):
    informatieobject = URLModelChoiceFilter(instance_path='canonical')
    
    class Meta:
        model = ObjectInformatieObject
        fields = ('object', 'informatieobject')
```

This:

1. Reduces the need for subclasses/extra registration
2. Makes it more explicit that the relation is being traversed (and thus allows you to see where to add `select_related` if/when needed)

The automatic filter generation from django-filters is really cool, but we shouldn't try to shoehorn everything in it. The vng-api-common subclasses exist to do "the right" thing by default for the common situation, however, with the versioning setup, this is no longer "a common situation" and we should treat it as such :smile: